### ISSUE TITLE
Aslice

### ISSUE THREAD

**Issue by emplumb** [Tynybeko/aslice_copy#1]
I was just wondering how you got a copy of the repo? Were you a developer for the site? And do you have any plans to continue it on in some way? I ask because I think it was a great idea and would love to see if it could be continued in some manner

**Comment by Tynybeko** [comment]
Yes, I participated in the project for a couple of days. While I didn’t
think about continuing it somehow, I left it as a template for studying the
structure of the project. Any suggestion?)


Сб, 19 окт. 2024 г. в 21:24, Evan Plumb ***@***.***>:

**Comment by emplumb** [comment]
well, now that's a very good question on how to proceed. My initial thought was to familiarize myself more to see if there was anything that could be simplified or removed in order to cut down on costs since that was the main reason for ending the site. I have no idea if there's anything feasibly that could be done, but perhaps you have some ideas?

I cloned it locally to poke around and ran through the setup steps but was getting an error, and haven't had time to look into it yet. 

edit: I was able to get the code to compile and hardcode my way through registration because I couldn't get past step 3 in the UI where you have to connect paypal or stripe haha

**Comment by emplumb** [comment]
I'm also seeing files and info committed that prob shouldn't unless everything has been deactivated already

**Comment by emplumb** [comment]
Also, do you happen to know what the biggest costs of running the site were?

**Comment by Tynybeko** [comment]
No I don’t know

Вс, 20 окт. 2024 г. в 22:55, Evan Plumb ***@***.***>:
